src/main/resources/assets/scripts/saml-redirect-auto-submit.js

```diff
@@ -0,0 +1,11 @@
+function initAutoSubmitSequence() {
+    document.forms[0].setAttribute("style", "display: none;");
+
+    window.setTimeout(function () {
+        document.forms[0].removeAttribute("style");
+    }, 5000);
+
+    document.forms[0].submit();
+}
+
+document.addEventListener('load', initAutoSubmitSequence());
```

Comment:
Pretty sure you want `initAutoSubmitSequence` without the brackets? Otherwise that's a function call that happens immediately and not on load.